src/test/tests/quickrecipes/how_to_start.py

```diff
@@ -0,0 +1,613 @@
+# ----------------------------------------------------------------------------
+#  CLASSES: nightly
+#
+#  Test Case:  basics.py
+#
+#  Mark C. Miller, Fri Aug 26 14:31:48 PDT 2022
+# ----------------------------------------------------------------------------
+#def LoadUtils(fname):
+#    with open(fname, 'r') as f: 
+#        prog = compile(''.join(f.readlines()),'visit_load_utils.err','exec')
+#        exec(prog)
+#
+#LoadUtils(tests_path('quickrecipes','vqr_utils.py'))
+
+def using_session_files():
+ 
+  try:
+    # using session files {
+    # Import a session file from the current working directory. 
+    RestoreSession('/home/juan/visit/visit.session', 0) 
+    # Now that VisIt has restored the session, animate through time.
+    for state in range(TimeSliderGetNStates()): 
+      TimeSliderSetState(state) 
+      SaveWindow() 
+    # using session files }
+    TestValueEQ('using session files error message',GetLastError(),'')
+    TestPOA('using session files no exceptions')
+  except:
+    TestFOA('using session files exception', LINE())
+    pass
+  vqr_cleanup()
+
+def getting_something_on_the_screen():
+
+  try:
+    # getting something on the screen {
+    # Step 1: Open a database 
+    OpenDatabase('~juanita/silo/stuff/wave.visit') 
+
+    # Step 2: Add plots with default properties
+    AddPlot('Pseudocolor', 'pressure') 
+    AddPlot('Mesh', 'quadmesh') 
+
+    # Step 3: Draw the plots with default view
+    DrawPlots() 
+
+    # Step 4: Iterate through time and save images 
+    for state in range(0,TimeSliderGetNStates(),10): 
+      TimeSliderSetState(state) 
+      SaveWindow() 
+    # getting something on the screen }
+    TestValueEQ('getting something on the screen error message',GetLastError(),'')
+    TestPOA('getting something on the screen no exceptions')
+  except:
+    TestFOA('getting something on the screen exception', LINE())
+    pass
+  vqr_cleanup()
+
+#
+# Get some utils shared across many .py files. Exec'ing in this way
```

Comment:
@cyrush can you please take a look at this code?

Ultimately, my goal is to share functionality in `vqr_utils.py` across multiple other `.py` test files here. This functionality includes overrides of some CLI funcs. Nothing else I tried worked. These 3 lines will have to be copied in each `.py` file but thats better than copying all of `vqr_utils.py` everywhere I will need it.

Reply:
@markcmiller86 I suggest instead we import `vqr_utils.py`

Add the path where it lives to the system path.

Then do something like:

from vqr_utils import `RestoreSession`, `OpenDatabase` and re-write those to be simple functions instead of overriding names:

```
def RestoreSession(f, vdirFlag=0):
    return visit.RestoreSession(vqr_path(f),vdirFlag)

def OpenDatabase(db, ti=0, pname=''):
    return visit.OpenDatabase(vqr_path(db),ti,pname)
```

let me know if this makes sense.

Reply:
Ok, lemme try that and see what I get.

Reply:
So, I am not able to get this to work. It gets close. But, I have code in `vqr_utils.py` using `tests_path()` and `silo_data_path()` from `visit_test_main.py`. If I don't import those, it excepts when I call `tests_path()`. If I do import those with `from visit_test_main import tests_path, silo_data_path`, it fails like so...

```
Traceback (most recent call last):
  File "/Users/miller86/visit/visit/src/test/tests/quickrecipes/how_to_start.py", line 9, in <module>
    from vqr_utils import vqr_path, vqr_cleanup
  File "/Users/miller86/visit/visit/src/test/tests/quickrecipes/vqr_utils.py", line 9, in <module>
    from visit_test_main import tests_path, silo_data_path
  File "/Users/miller86/visit/visit/build/lib/site-packages/visit_testing/visit_test_main.py", line 69, in <module>
    sys.path.append(os.path.abspath(os.path.split(__visit_script_file__)[0]))
NameError: name '__visit_script_file__' is not defined
```

which is a problem with the `import visit_test_main` which I think is expecting whatever imports it to have already defined `__visit_script_file__`

Reply:
Maybe the `tests_path` and `silo_data_path` need to be something other than `visit_test_main` for it to work.